tutorials/advanced-development/01-advanced-pieces-topics/01-ajax-pieces.md

```diff
@@ -0,0 +1,134 @@
+---
+title: AJAX Features: Enhanced Browsing for Pieces
+layout: tutorial
+---
+
+# AJAX Features: Enhanced Browsing for Pieces
+
+Asynchronous JavaScript and XML, commonly known as [AJAX](https://api.jquery.com/jquery.ajax/), provides techniques for more dynamic and responsive websites. You can use AJAX with Pieces to refresh and display content without fully refreshing the page. To demonstrate this, we'll work off of the `people-pages` example from the [Reusable Content with Pieces](../../core-concepts/03-reusable-content-pieces/README.md) section, but you should be able to easily adapt this example to your own project.
```

Comment:
Link directly to the subpage about people-pages and call out that you're talking about pieces-pages here. The main complaint we get is insufficiently clear context.